lib/src/screens/ionia/cards/ionia_more_options_page.dart

```diff
@@ -27,40 +26,46 @@ class IoniaMoreOptionsPage extends BasePage {
       padding: const EdgeInsets.all(16.0),
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.stretch,
-          children: [
-            SizedBox(height: 10,),
-            Center(child: Text(S.of(context).choose_from_available_options, style: textMedium(
-              color:  Theme.of(context).primaryTextTheme!.headline6!.color!,
-            ),)),
-            SizedBox(height: 40,),
-            InkWell(
-              onTap: () async {
-                final amount = await  Navigator.of(context).pushNamed(Routes.ioniaCustomRedeemPage, arguments: [giftCard]) as String;
-                if(amount.isNotEmpty){
-                  Navigator.pop(context, amount);
-                }
-              },
-              child: _GradiantContainer(
-                content:  Padding(
-                        padding: const EdgeInsets.only(top: 24, left: 20, right: 24, bottom: 50),
-                        child: Text(
-                          S.of(context).custom_redeem_amount,
-                          style: textXLargeSemiBold(),
-                        ),
-                      ),
-                    ),
-            )
-              ],    
-        ),
+        children: [
+          SizedBox(
+            height: 10,
+          ),
+          Center(
+              child: Text(
+            S.of(context).choose_from_available_options,
+            style: textMedium(
+              color: Theme.of(context).primaryTextTheme.headline6!.color!,
+            ),
+          )),
```

Comment:
**Minor:**
Fix format and remove un-necessary `,` from `SizedBox` so it can take only 1 line